install.md

```diff
@@ -220,6 +220,18 @@ make BUILDTAGS='seccomp apparmor'
 | selinux   | selinux process and mount labeling | libselinux  |
 | apparmor  | apparmor profile support           | libapparmor |
 
+### Vendoring - Dependency Management
+
+This project is using [vndr](https://github.com/LK4D4/vndr) for managing dependencies, which is a tedious and error-prone task. Doing it manually is likely to cause inconsistencies between the `./vendor` directory (i.e., the downloaded dependencies), the source code that imports those dependencies and the `vendor.conf` configuration file that describes which packages in which version (e.g., a release or git commit) are a dependency.
+
+To ease updating dependencies, we provide the `make vendor` make target, which fetches all dependencies mentioned in the `vendor.conf`.  `make vendor` whitelists certain packages to prevent the `vndr` tool from removing packages that the tests suite (see `./test`) imports.
```

Comment:
Unnecessary second "make" in "make vendor make target"; unnecessary "the" in "the vendor.conf"; "tests suite" reads better as singular

Reply:
Thanks!